babel.config.js

```diff
@@ -21,4 +23,22 @@ module.exports = {
     'angularjs-annotate',
     isDevelopment && 'react-refresh/babel',
   ]),
+  env: {
+    production: {
+      plugins: compact([
```

Comment:
`compact` can be removed here — it can be used to remove plugins from the list conditionally but there are not conditions here.

Reply:
:+1: removed in 965bf935c914ff98b99a97be97115ddd2ab20e82

_note: Seems like we could move all the environment conditions into the [env-option](https://babeljs.io/docs/en/6.26.3/babelrc#env-option) as Nick suggested [above](https://github.com/Trustroots/trustroots/pull/1291#issuecomment-599250830). Not in this PR._